app/build.gradle

```diff
@@ -163,60 +170,55 @@ android {
 }
 
 dependencies {
-    def supportVersion = "28.0.0"
-    def playServicesVersion = "16.0.0"
-    def daggerVersion = "2.15"
-    def okHttpVersion = "3.10.0"
-    def retrofitVersion = "2.4.0"
-    def butterKnifeVersion = "8.8.1"
-    implementation 'com.android.support:multidex:1.0.3'
-    implementation "com.android.support:support-v4:$supportVersion"
-    implementation "com.android.support:support-annotations:${supportVersion}"
-    implementation "com.android.support:cardview-v7:${supportVersion}"
-    implementation "com.android.support:recyclerview-v7:${supportVersion}"
-    implementation "com.android.support:appcompat-v7:${supportVersion}"
-    implementation "com.android.support:design:${supportVersion}"
-    implementation "com.android.support:customtabs:${supportVersion}"
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
-    implementation "com.google.android.gms:play-services-maps:${playServicesVersion}"
-    implementation "com.google.android.gms:play-services-location:${playServicesVersion}"
-    implementation "com.google.android.gms:play-services-gcm:${playServicesVersion}"
-    implementation "com.google.android.gms:play-services-places:${playServicesVersion}"
+    implementation 'androidx.multidex:multidex:2.0.1'
```

Comment:
I think we should extract all dependencies versions and category like on this article :
https://proandroiddev.com/treat-your-build-files-as-you-treat-your-code-def80ac52978

It will be easier to understand and make changes

Reply:
We'll do it later! Unless you want to do it on another branch...